src/features/landing/components/Tabs.tsx

```diff
@@ -1,32 +1,15 @@
 /** @jsxImportSource preact */
-import h from "preact"
 import { useState } from "preact/hooks"
 import { clsx } from "~/lib"
 import tabs from "./Tabs.module.css"
 import { ProductCard } from "./ProductCard"
-import solanaIcon from "../assets/solana-chains.svg"
-import ethereumIcon from "../assets/evm-chains.svg"
-import { evmProducts, solanaProducts } from "../data"
+import { evmProducts } from "../data"
 
 export const Tabs = () => {
-  const [tabId, setTabId] = useState("evm-products")
+  const [tabId] = useState("evm-products")
```

Comment:
If we don't have the tabs we can remove the useState logic

Reply:
I was thinking I should leave it for the future if we need the extra capability, but we can always come back and get it again later. I'll update this.

Reply:
Removed